### ISSUE TITLE
Run branch coverage as codecov does using tox

### ISSUE THREAD

**Issue by lekhajee** [rackerlabs/mimic#369]
Currently the test coverage running on tox is not the same as the one on codecov. It would be nice if we could get the same kind of feedback as codecov locally so we don't have to wait for travis builds to finish to realize we are missing some test coverage.

**Comment by glyph** [comment]
Is isn't? The `coverage report` in tox has been showing sub-100% coverage since the beginning, because it *is* the same as the one on codecov; or so I thought.

**Comment by lekhajee** [comment]
yes, but it does not indicate the lines missing branch coverage, which is a bummer.

**Comment by cyli** [comment]
We can probably add a directive to do `coverage html` (and generate an HTML coverage report for every environment that tox is running on) as well as `coverage report -m`

**Comment by glyph** [comment]
I don't know if `tox` is the right place to do the `coverage html`, because we would ideally want to unify the coverage from multiple environments.  Perhaps we could have a script?
